plugins/modules/zos_copy.py

```diff
@@ -2007,9 +2030,13 @@ def allocate_destination_data_set(
         return False, dest_params
 
     # Giving more priority to the parameters given by the user.
+    # Cover case the user set executable to true to create dataset valid.
     if dest_data_set:
         dest_params = dest_data_set
         dest_params["name"] = dest
+        if executable:
```

Comment:
code structure was changed to address `executable=true` changes later on in the conditional block structure to avoid overriding user-provided attrs for destination data set. Marking this conversation as resolved